x-pack/plugins/alerting/server/saved_objects/migrations.ts

```diff
@@ -85,11 +85,18 @@ export function getMigrations(
     pipeMigrations(removeNullsFromSecurityRules)
   );
 
+  const migrationSecurityRules714 = createEsoMigration(
+    encryptedSavedObjects,
+    (doc): doc is SavedObjectUnsanitizedDoc<RawAlert> => isSecuritySolutionRule(doc),
+    pipeMigrations(removeNullAuthorFromSecurityRules)
+  );
+
   return {
     '7.10.0': executeMigrationWithErrorHandling(migrationWhenRBACWasIntroduced, '7.10.0'),
     '7.11.0': executeMigrationWithErrorHandling(migrationAlertUpdatedAtAndNotifyWhen, '7.11.0'),
     '7.11.2': executeMigrationWithErrorHandling(migrationActions7112, '7.11.2'),
     '7.13.0': executeMigrationWithErrorHandling(migrationSecurityRules713, '7.13.0'),
+    '7.14.0': executeMigrationWithErrorHandling(migrationSecurityRules714, '7.13.0'),
```

Comment:
```suggestion
    '7.14.0': executeMigrationWithErrorHandling(migrationSecurityRules714, '7.14.0'),
```

Reply:
Thanks, fixed that part. We are targeting `7.14.1` now instead of `7.14.0`